src/SIMD.jl

```diff
@@ -25,18 +25,18 @@ for sz in (8, 16, 32, 64, 128)
 
         Base.convert(::Type{Bool}, b::$Boolsz) = b.int != 0
 
-        Base. ~(b::$Boolsz) = $Boolsz(~b.int)
-        Base. !(b::$Boolsz) = ~b
-        Base. &(b1::$Boolsz, b2::$Boolsz) = $Boolsz(b1.int & b2.int)
-        Base. |(b1::$Boolsz, b2::$Boolsz) = $Boolsz(b1.int | b2.int)
+        Base.:~(b::$Boolsz) = $Boolsz(~b.int)
+        Base.:!(b::$Boolsz) = ~b
+        Base.:&(b1::$Boolsz, b2::$Boolsz) = $Boolsz(b1.int & b2.int)
+        Base.:|(b1::$Boolsz, b2::$Boolsz) = $Boolsz(b1.int | b2.int)
```

Comment:
These are inside the comment so I wasn't sure if it's better to change them or leave them as-is. Let me know if I need to revert them (I'll remove this from the patch and then force-push).